src/javascript/_common/auth.js

```diff
@@ -78,9 +78,8 @@ export const isOAuth2Enabled = () => {
     return false;
 };
 
-export const getLogoutHandler = onWSLogoutAndRedirect => {
-    const oAuth2Logout = OAuth2Logout(onWSLogoutAndRedirect);
-    return oAuth2Logout;
+export const requestOauth2Logout = onWSLogoutAndRedirect => {
+    OAuth2Logout(onWSLogoutAndRedirect);
```

Comment:
`Oauth2Logout` does not return a callback anymore, refer:
https://github.com/deriv-com/auth-client/blob/995ba957d724e247ac5d6e87656813beccaa1426/src/oidc/oidc.ts#L271